completions/fish/crio.fish

```diff
@@ -10,7 +10,7 @@ function __fish_crio_no_subcommand --description 'Test if there has been any sub
 end
 
 complete -c crio -n '__fish_crio_no_subcommand' -f -l additional-devices -r -d 'Devices to add to the containers '
-complete -c crio -n '__fish_crio_no_subcommand' -f -l apparmor-profile -r -d 'Name of the apparmor profile to be used as the runtime\'s default. The default profile name is \'crio-default-\' followed by the version string of CRI-O. (default: "crio-default-1.17.0-dev"). This only takes effect if the user does not specify a profile via the Kubernetes Pod\'s metadata annotation.'
+complete -c crio -n '__fish_crio_no_subcommand' -f -l apparmor-profile -r -d 'Name of the apparmor profile to be used as the runtime\'s default. The default profile name is \'crio-default-\' followed by the version string of CRI-O. (default: "crio-default-v1.17.0-rc1-dev"). This only takes effect if the user does not specify a profile via the Kubernetes Pod\'s metadata annotation.'
```

Comment:
I thinks its a major drawback that we include the version in the profile. It changes per release and opens-up a common risk of failure when switching to a new version. Maybe we can change that to just run a default profile name. The profile is also not very frequently changing...